string start_time
string end_time
---
string iso_duration

